parity/configuration.rs

```diff
@@ -159,13 +166,14 @@ impl Configuration {
         let experimental_rpcs = self.args.flag_jsonrpc_experimental;
         let secretstore_conf = self.secretstore_config()?;
         let format = self.format()?;
+        let metrics_conf = self.metrics_config()?;
         let keys_iterations = NonZeroU32::new(self.args.arg_keys_iterations)
             .ok_or_else(|| "--keys-iterations must be non-zero")?;
 
         let cmd = if self.args.flag_version {
             Cmd::Version
         } else if self.args.cmd_signer {
-            let authfile = ::signer::codes_path(&ws_conf.signer_path);
+            let authfile = crate::signer::codes_path(&ws_conf.signer_path);
```

Comment:
Is this related to Prometheus?

Reply:
The answer is no, but it is just syntax change needed fo the rust 2018 edition.

Reply:
Oh, yes, I do not know that I was thinking... Lol